digdag-cli/src/main/java/io/digdag/cli/Server.java

```diff
@@ -134,7 +135,7 @@ private void startServer()
         Properties props = buildServerProperties();
         ConfigElement ce = PropertyUtils.toConfigElement(props);
         ServerConfig serverConfig = ServerConfig.convertFrom(ce);
-        ServerBootstrap.start(buildServerBootstrap(version, serverConfig));
+        ServerBootstrap.start(buildServerBootstrap(version, serverConfig, loadSystemPlugins(props)));
```

Comment:
In master branch, initialize ServerBootstrap class directly.
But use buildServerBootstrap method to override from `digdag-tests/src/test/java/acceptance/AccessControllerIT.java`